### ISSUE TITLE
Primary buttons have insufficient contrast

### ISSUE THREAD

**Issue by anthonydillon** [ubuntudesign/vanilla-framework#373]
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.65:1. Recommendation: change background to #d6410d.

**Comment by barrymcgee** [comment]
This recommendation (ditto #374) will require confirmation from @yaili - one of the aims of v1 is to neutralise any brand leanings, so it may make sense to address both here?

**Comment by yaili** [comment]
@anthonydillon What conformance level?

**Comment by anthonydillon** [comment]
So they do not fail a standard contrast test.

**Comment by barrymcgee** [comment]
Will be addressed by #397
